Code/src/main/resources/ui/style.css

```diff
@@ -7,95 +7,173 @@ CustomUIPane {
+}
+
+.value.block {
+	-fx-padding: 0 20 0 20;
+    -fx-background-color: #BD1550;
+}
+
+.selected .display.block {
+	-fx-background-color: #6E7C0C;
+	-fx-effect: dropshadow(three-pass-box, rgba(0,0,0,0.8), 10, 0, 0, 0);
+}
+
+.display.block {
+    -fx-background-color: #8A9B0F;
+    -fx-text-alignment: center;
+}
+
+.graph.block {
+    -fx-background-color: #ecf0f1;
 }
 
-.function.block .description {
+.menu.block {
     -fx-font-size: 12px;
-    -fx-padding: 0;
+    -fx-background-color: #490A3D;
 }
 
-.function.block .nestSpace {
-    -fx-padding: 15;
+.menu.block .title {
+    -fx-font-size: 15px;
+    -fx-font-weight: bold;
+    -fx-font-color: #FCEBB6;
+    -fx-background-color: #A4849E;	
 }
 
-.function.block #argumentSpace Label {
-    -fx-padding: 0 5 0 5;
-    -fx-font-size: 14px;
-    -fx-font-weight: none;
-    -fx-background-color:white;
-    -fx-border-color: black;
-    -fx-border-width: 1;
+.menu.block .title:hover {
+    -fx-font-size: 15px;
+    -fx-font-weight: bold;
+    -fx-font-color: #FCEBB6;
+    -fx-background-color: #926C8B;
 }
 
-.function.block #argumentSpace Label.error {
-    -fx-background-color:pink;
-    -fx-border-color: red;
+
+
+.menu.block .title .arrow-button .arrow{
+    -fx-background-color: #EDE6EC;
+    -fx-focus-color: white;
 }
 
-.function.block #argumentSpace Label.result {
-    -fx-background-color: black;
-    -fx-text-fill: white;
+.menu.block .content {
+    -fx-font-size: 15px;
+    -fx-font-weight: normal;
 }
 
-.value.block {
-    -fx-background-color: #8e44ad;
+.menu.block Button {
+	-fx-background-color: #A4849E;
 }
 
-.display.block {
-    -fx-background-color: #c0392b;
+.menu.block Button:hover {
+	-fx-background-color: #926C8B;
 }
 
-.graph.block {
-    -fx-background-color: #ecf0f1;
+.menu.block #categorySpace TitledPane {
+    -fx-padding: 3px;  
 }
 
-.value.block .title, .display.block .title {
+.menu.block #utilSpace {
+    -fx-padding: 3px;
+    -fx-spacing: 6px;  
+}
 
+.menu.block .list-cell:even {
+	-fx-background-color: #DBCED8;	
 }
 
-.block .title {
-    -fx-text-fill: #ecf0f1;
+.menu.block .list-cell:odd {
+	-fx-background-color: #C8B6C5;	
+}
+
+.argumentSpace {
+}
+
+.argumentSpace Label.leftArgument {
+    -fx-padding: 0 5 0 5;
+    -fx-font-family: Titillium;
+    -fx-text-fill: black;
+    -fx-font-size: 14px;
+    -fx-font-weight: normal;
+    -fx-background-color: white;
+    -fx-border-width: 1;
+}
+
+.argumentSpace Label.rightArgument {
+    -fx-padding: 0 5 0 5;
+    -fx-font-family: Titillium;
+    -fx-font-size: 14px;
+    -fx-font-weight: normal;
+    -fx-text-fill: white;
+    -fx-background-color: black;
+    -fx-border-width: 2;
+}
+
+.argumentSpace .error Label.leftArgument {
+    -fx-background-color:red;
+    -fx-border-color: red;
+    -fx-text-fill: white;
 }
 
 .connection {
     -fx-stroke-width: 3px;
-    -fx-stroke: white;
+    -fx-stroke: #FCEBB6;
     -fx-fill: null;
 }
 
 .connection.error {
-    -fx-stroke: orange;
+    -fx-stroke: red;
```

Comment:
Currently I'd say red stands better out against the also orange function blocks. Leaving it red, could discuss this more some other time.